paper/paper.md

```diff
@@ -0,0 +1,176 @@
+
+AutoEmulate can additionally search over different data preprocessing methods, such as normalization or dimensionality reduction techniques. AutoEmulate implements principal component analysis (PCA) and variational autoencoders (VAEs) for handling high dimensional input or output data. For example, the following code compares three different output transformations: no transformation, PCA with 16 components, and PCA with 32 components in combination with the default set of emulators:
+
+```python
+from autoemulate.transforms import PCATransform
+
+ae = AutoEmulate(
+    x, 
+    y,
+    y_transforms_list=[
+      [],
+      [PCATransform(n_components=16)],
+      [PCATransform(n_components=32)]
+    ],
+)
+```
+
+The result in this case will return the best combination of model and output transformation. The figure below shows an example result of fitting a Gaussian Process emulator in combination with PCA to a reaction-diffusion simulation (see the full [tutorial](https://alan-turing-institute.github.io/autoemulate/tutorials/emulation/02_dim_reduction.html) for a detailed overview).
+
+![GP with PCA emulator prediction for a reaction diffusion simulation compared to the ground truth.](reaction_diffusion_emulation.png)
+
+Once an emulator has been trained it can be used to generate fast predictions for new input values or to perform [downstream tasks](https://alan-turing-institute.github.io/autoemulate/tutorials/tasks/index.html) such as sensitivity analysis or model calibration. For example, to run Sobol sensitivity analysis one only needs to pass the trained emulator and some information about the data. Below is a dummy example assuming a simulation with two input parameters `param1` and `param2`, each with a plausible range of values, and two outputs `output1` and `output2`:
+
+```python
+from autoemulate.core.sensitivity_analysis import SensitivityAnalysis
+
+input_parameters_ranges = {
+    'param1': (0, 1),
+    'param2': (0, 10),
+}
+
+problem = {
+    'num_vars': 2,
+    'names': ["param1", "param2"],  
+    'bounds': input_parameters_ranges.values(),
+    'output_names': ["output1", "output2"],
+}
+
+sa = SensitivityAnalysis(emulator, problem=problem)
+sobol_df = sa.run()
+```
+
+The PyTorch backend enables fast Bayesian model calibration using gradient-based inference methods such as Hamiltonian Monte Carlo with Pyro. AutoEmulate provides a simple interface for this given a trained PyTorch emulator, input parameter ranges (same as in the sensitivity analysis example), and real-world observations:
+
+```python
+from autoemulate.calibration.bayes import BayesianCalibration
+
+observations = {'output1': 0.5, 'output2': 7.2}
+
+bc = BayesianCalibration(
+    emulator, 
+    input_parameters_ranges, 
+    observations, 
+)
+mcmc = bc.run()
+```
+
```

Comment:
It might be worth linking directly to a more applied example with a sentence something like this after the code? As above though, this level of detail may be unncessary given the link is already included above for downstream tasks.
```suggestion
A more complete application of Bayesian model calibration to simulating an epidemic is demonstrated [here](https://alan-turing-institute.github.io/autoemulate/tutorials/tasks/03_bayes_calibration.html).
```